### PR TITLE
Deprecated the `<re>` element

### PR DESCRIPTION
**Pull request by trishaoconnor**
Deprecated the `<re>` element for #2487 and in examples replaced `<re>` with `<entry>` element with the `@type` set to "relatedEntry".

### CODE REVIEW

**Comment by laurentromary** [comment]
I have checked this pull request and it all seems like we want it. Thanks!

File: P5/Source/Specs/re.xml [hunk 1]
```diff
@@ -7,26 +7,34 @@ $Date$
 $Id$
 -->
 <?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
-<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="dictionaries" ident="re">
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="dictionaries" ident="re"
+  validUntil="2026-03-10">
+  <desc xml:lang="en" type="deprecationInfo" versionDate="2023-01-10"> Because an <gi>entry</gi> can
+    now occur inside an <gi>entry</gi>, the <gi>re</gi> element is no longer needed. These
+    Guidelines recommend using an <gi>entry</gi> with a <att>type</att> attribute value of
+      <val>related</val> instead.</desc>
```
Comment (HelenaSabel) [inline P5/Source/Specs/re.xml]:
```suggestion
      <val>relatedEntry</val> instead.</desc>
```

**Review by HelenaSabel — CHANGES_REQUESTED** [review]
My first comment concerns commit 61247e54d5412e5f1a2e46adb76ef92879c66b3a: I think it might have been included in this PR by mistake.

Then I made a suggestion so that the recommended value for `@type` in the deprecation note matches the examples. @trishaoconnor: if you agree you can click on “commit suggestion” :)

**Comment by trishaoconnor** [comment]
Thank you @HelenaSabel for spotting these mistakes. My apologies for not recognising that the commit was added to this PR. That was an accident.  I have clicked "commit suggestion" so the deprecation note now matches the examples.

**Comment by HelenaSabel** [comment]
@trishaoconnor: do you want to try to delete commit https://github.com/TEIC/TEI/commit/61247e54d5412e5f1a2e46adb76ef92879c66b3a? I think you could try:
```
git revert --strategy resolve 61247e54d5412e5f1a2e46adb76ef92879c66b3a
```
That will delete the commit (without altering the history). But if @sydb, who is assigned to this PR, thinks it’s ok to merge it as it is, fine with me :)

**Comment by sydb** [comment]
Yes, the global addition of `mode="add"` was added to this PR by mistake. But since it is a separate commit, and has no effect on anything anyway, I would just not worry about it. (Except to _avoid_ a “squash and merge”.)
But that may just be me being lazy — feel free to separate to a separate PR if y’all think it important.

**Comment by sydb** [comment]
An attempt to ping @GusRiva on the TEI repo. Please ignore.

**Review by HelenaSabel — APPROVED** [review]

**Pull request merged**